core/alert.py

```diff
@@ -82,12 +83,8 @@ def info(content, log_in_file=None, mode=None, event=None, language=None, thread
         None
     """
     if is_not_run_from_api():  # prevent to stdout if run from API
-        if pyversion is 2:
-            sys.stdout.write(color.color('yellow') + '[+] ' + color.color('green') +
-                             content.encode('utf8') + color.color('reset') + '\n')
-        else:
-            sys.stdout.buffer.write(bytes(color.color('yellow') + '[+] ' + color.color('green') +
-                                          content + color.color('reset') + '\n', 'utf8'))
+        sys.stdout.write(color.color('yellow') + '[+] ' + color.color('green') +
+                six.text_type(content) + color.color('reset') + '\n')
```

Comment:
Using of six in python2 and 3 is best way I found. It automatically converts the text type to the version currentlly in use.
So using dir_scan, I was facing some problems in printing of info() in python3 (results were coming but info() was not able to print, No issues with python2). So using six.text_type() worked well.